modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -148,4 +182,92 @@ private String getFullClassName(Element classElement) {
         }
         return className.toString() + classElement.getSimpleName();
     }
+
+    private String getMethodIdentifierString(Element methodElement) {
+        TypeVisitor<ExecutableType, Void> executableTypeVisitor =
+            new SimpleTypeVisitor6<ExecutableType, Void>() {
+                @Override
+                public ExecutableType visitExecutable(
+                    ExecutableType executableType, Void obj
+                ) {
+                    return executableType;
+                }
+            };
+        List<? extends TypeMirror> methodParams =
+            methodElement.asType()
+                .accept(executableTypeVisitor, null)
+                .getParameterTypes();
+        String fullClassPattern =
+            getClassHeader(methodElement.getEnclosingElement());
+        String fullClassName =
+            fullClassPattern.substring(0, fullClassPattern.indexOf('('));
+        String methodSignature = "";
+        if (methodElement.getKind().equals(ElementKind.CONSTRUCTOR)) {
+            int index = fullClassName.lastIndexOf("\\$");
+            if (index != -1) {
+                methodSignature +=
+                    "L" + fullClassName.substring(0, index)
+                        .replace("\\$", "/") + ";";
+            }
+        }
+        methodSignature += getMethodSignature(methodParams);
+        return fullClassName + "," +
+            methodElement.getSimpleName().toString() + "," +
+            methodSignature.replace("\\$", "$");
+    }
+
+    private String getMethodSignature(List<? extends TypeMirror> methodParams) {
+        if (methodParams.isEmpty()) {
+            return "";
+        }
+        String methodSignature = "";
```

Comment:
I think it would make sense to use a `StringBuilder` here. It's generally something that you'd want to use when the concatenation is happening inside a loop in order to avoid a lot of string copying.